aten/src/ATen/cuda/tunable/GemmHipblaslt.h

```diff
@@ -305,7 +466,7 @@ class HipblasltGemmOp : public Callable<ParamsT> {
             workspace_size,
             at::cuda::getCurrentCUDAStream()));
 
-      TORCH_HIPBLASLT_CHECK(hipblasLtMatmulDescDestroy(matmul));
+      //TORCH_HIPBLASLT_CHECK(hipblasLtMatmulDescDestroy(matmul));
```

Comment:
Why is this line commented out now? We still need to destruct HipBlasLtMatmulDescriptor ?

Reply:
matmul is an instance of HipBlasLtMatmulDescriptor which has an auto-destruct.  I should have deleted the line instead of just commenting it out.

Longer term, I need to move the various CuBlasLt*Descriptor classes into their own header so they can be reused here, but I made one attempt and it caused unexpected mem access faults so I will do that in a follow-up PR.

These *Descriptor classes are here, with a TODO to move them from the original author: https://github.com/pytorch/pytorch/blob/5a15cbfa440e87f8439d1f28b3d8df0c51daf088/aten/src/ATen/cuda/CUDABlas.cpp#L836-L913